test/tokenomics/liquidityGaugeV5.ts

```diff
@@ -138,10 +155,10 @@ describe("Liquidity Gauge V5", () => {
       await increaseTimestamp(86400)
```

Comment:
Using `increaseTimestamp()` intentionally here?

Reply:
Changed to setTimestamp